src/Symfony/Component/Routing/Loader/YamlFileLoader.php

```diff
@@ -199,25 +221,26 @@ protected function validate($config, $name, $path)
             ));
         }
         if (isset($config['resource']) && isset($config['path'])) {
-            throw new \InvalidArgumentException(sprintf(
-                'The routing file "%s" must not specify both the "resource" key and the "path" key for "%s". Choose between an import and a route definition.',
-                $path, $name
-            ));
+            throw new \InvalidArgumentException(sprintf('The routing file "%s" must not specify both the "resource" key and the "path" key for "%s". Choose between an import and a route definition.', $path, $name));
```

Comment:
I feel like may have mislead you sorry: if there are no changes here, the CS should be kept (same below)